README.md

```diff
@@ -296,6 +296,43 @@ $response = $matej->request()->recommendation($recommendation)->send();
 
 Model names will be provided to you by LMC.
 
+## Forgetting user data (GDPR)
+Matej can "forget" user data, either by anonymizing or by deleting them. The right to forget is part of [data protection in the European Union](https://ec.europa.eu/info/law/law-topic/data-protection_en),
```

Comment:
It is officially called "Right to erasure (‘right to be forgotten’)" https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:32016R0679#d1e2606-1-1

So we should follow the same name for it.